f8a_worker/workers/recommender.py

```diff
@@ -514,9 +514,13 @@ class RecommendationV2Task(BaseTask):
 
     def call_pgm(self, payload):
         """Calls the PGM model with the normalized manifest information to get the relevant packages"""
-        pgm_url = PGM_URL_REST + "/api/v1/schemas/kronos_scoring"
         try:
-            if payload is not None:
+            # TODO remove hardcodedness for payloads with multiple ecosystems
```

Comment:
We need to handle multiple ecosystem scenario. Currently we handle multiple pom, but we may not be able to handle pom with requirements.txt with this approach